docs/02-guides/display-wysiwyg-content.mdx

```diff
@@ -0,0 +1,344 @@
+---
+title: Display WYSIWYG content
+description:
+  Let your users write their content without needing any HTML or React knowledge
+  while still being able to deliver a qualitative UI (performant, responsive,
+  accessible…) to customers.
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import ContactLink from "@site/src/components/ContactLink";
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<SinceVersion tag="3.4" />
+
+<p>{frontMatter.description}</p>
+
+WYSIWYG stands for **W**hat **Y**ou **S**ee **I**s **W**hat **Y**ou **G**et. It
+means that your back office users write their content without needing any HTML
+or React knowledge. This is the case in most CMS tools nowadays. The output
+though is usually HTML which does not necessarily match the React components
+you've built in your Front-Commerce application.
+
+This is why we've built a `theme/modules/Wysiwyg` component in Front-Commerce.
+In this guide you will learn how to use it and how to customize its display.
+
+## `<Wysiwyg />` usage
+
+Here is an example of using this component:
+
+```tsx
+import React from "react";
+import Wysiwyg from "theme/modules/Wysiwyg";
+
+export default (props) => <Wysiwyg content={props.cms.contentWysiwyg} />;
+```
+
+The `contentWysiwyg` property must come from a GraphQL field of type `Wysiwyg`.
+It comes from the `WysiwygFragment` fragment available at
+[`theme/modules/Wysiwyg/WysiwygFragment.gql`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/3.4.0/packages/theme-chocolatine/theme/modules/Wysiwyg/WysiwygFragment.gql),
+and using in `CmsPageFragment` available at
+[`theme/pages/CmsPage/CmsPageFragment.gql`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/3.4.0/packages/theme-chocolatine/theme/pages/CmsPage/CmsPageFragment.gql).
```

Comment:
The wording is quite weird here, here is my attempt at re-phrasing it, but the part for `CmsPageFragment` also needs some improvement.

```suggestion
The `contentWysiwyg` property originates from a GraphQL field `Wysiwyg`.
To see the required fields, see the `WysiwygFragment` fragment  reference available at
[`theme/modules/Wysiwyg/WysiwygFragment.gql`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/3.4.0/packages/theme-chocolatine/theme/modules/Wysiwyg/WysiwygFragment.gql),
and using in `CmsPageFragment` available at
[`theme/pages/CmsPage/CmsPageFragment.gql`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/3.4.0/packages/theme-chocolatine/theme/pages/CmsPage/CmsPageFragment.gql).
```